docs/docs/creating-a-source-plugin.md

```diff
@@ -2,62 +2,29 @@
 title: Creating a Source Plugin
 ---
 
-There are two types of plugins that work within Gatsby's data system: "source"
-and "transformer" plugins.
+Source plugins are essentially out of the box integrations between Gatsby and various third-party systems. 
 
-- **Source** plugins "source" data from remote or local locations into what
-  Gatsby calls [nodes](/docs/node-interface/).
-- **Transformer** plugins "transform" data provided by source plugins into new
-  nodes and/or node fields.
+These systems can be CMSs like Contentful or Wordpress, other cloud services like Lever and Strava, or your local filesystem -- literally anything that has an API. Currently, Gatsby has [over 300 source plugins](/plugins/?=gatsby-source).
```

Comment:
somehow i like the 2 points you deleted more.

i suggest links if someone not know the services
```suggestion
These systems can be CMSs like [Contentful](https://www.gatsbyjs.org/packages/gatsby-source-contentful/?=source%20Contentful) or [Wordpress](https://www.gatsbyjs.org/packages/gatsby-source-contentful/?=source%20Wordpress), other cloud services like [Lever](https://www.gatsbyjs.org/packages/gatsby-source-strava/?=gatsby-source-lever) and [Strava](https://www.gatsbyjs.org/packages/gatsby-source-strava/?=strava), or your local filesystem -- literally anything that has an API. Currently, Gatsby has [over 300 source plugins](/plugins/?=gatsby-source).
```

Reply:
These links are not correct

Reply:
yes - the wordpress link should point to https://www.gatsbyjs.org/packages/gatsby-source-wordpress/?=source%20Wordpress

Reply:
instead if pointing to strava.com the idea was to show that already there is a plugin for them